components/list/demo/list-color-indicator.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script type="module">
+			import '../../button/button-icon.js';
+			import '../../demo/demo-page.js';
+			import '../../dropdown/dropdown-menu.js';
+			import '../../dropdown/dropdown-more.js';
+			import '../../menu/menu.js';
+			import '../../menu/menu-item.js';
+			import '../list-item-content.js';
+			import '../list-item.js';
+			import '../list.js';
+		</script>
+	</head>
+	<body unresolved>
+
+		<d2l-demo-page page-title="d2l-list Color Indicator">
+
+			<h2>Basic List</h2>
+
+			<d2l-demo-snippet code-view-hidden>
```

Comment:
Is there a reason to hide the code-view in these demos? I'm not totally sure but I think we might just hide the code view when it contains test components that aren't relevant to the consumer, or when the code-view might interfere with the demo (floating workflow buttons?).

Reply:
It was mostly just for development purposes since there was a lot of code, but I'll remove this property.

Reply:
If you think it's worthwhile, especially for the huge nested demos, that's totally 😎 .